setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="data-simulator",
+    version="0.1",
```

Comment:
use `x.x.x`

Reply:
Fixed in e6bd4c559e8de5f1b2fe8b05faa47c8103c54e6f